data/pkgconfig/carla-host-plugin.pc

```diff
@@ -1,9 +1,10 @@
 prefix=X-PREFIX-X
-libdir=X-LIBDIR-X/carla
-includedir=X-INCLUDEDIR-X/carla
+libdir=X-LIBDIR-X
+includedir=X-INCLUDEDIR-X
+carla_libdir=${libdir}/carla
```

Comment:
I am adding a separate variable here because Carla installs binaries there that I want, so having a specific variable to get the path is useful